infer/modules/vc/modules.py

```diff
@@ -82,18 +84,12 @@ def get_vc(self, sid, *to_return_protect):
                 del self.net_g, self.cpt
                 if torch.cuda.is_available():
                     torch.cuda.empty_cache()
+                elif torch.backends.mps.is_available():
+                    torch.mps.empty_cache()
             return (
                 {"visible": False, "__type__": "update"},
-                {
-                    "visible": True,
-                    "value": to_return_protect0,
-                    "__type__": "update",
-                },
-                {
-                    "visible": True,
-                    "value": to_return_protect1,
-                    "__type__": "update",
-                },
+                to_return_protect0,
```

Comment:
Are these modifications necessary? 😂

Reply:
Yes because the value of protect would be dict.
It occurs when trying to infer again after unloading the model

```bash
  File "/Users/ftps/Retrieval-based-Voice-Conversion-WebUI/infer/modules/vc/modules.py", line 192, in vc_single
    audio_opt = self.pipeline.pipeline(
  File "/Users/ftps/Retrieval-based-Voice-Conversion-WebUI/infer/modules/vc/pipeline.py", line 430, in pipeline
    self.vc(
  File "/Users/ftps/Retrieval-based-Voice-Conversion-WebUI/infer/modules/vc/pipeline.py", line 239, in vc
    if protect < 0.5 and pitch is not None and pitchf is not None:
TypeError: '<' not supported between instances of 'dict' and 'float'
```

Reply:
To be precise, this causes the protect value to be `{'visible': True, 'value': 0.33, '__type__': 'update'}` instead of float.

Reply:
OK.